libs/wingc/src/lsp/completions.rs

```diff
@@ -716,27 +716,21 @@ fn get_completions_from_namespace(
 ) -> Vec<CompletionItem> {
 	// If a namespace has a class named "Util", then its members can be accessed directly from
 	// the namespace as a syntactic sugar. e.g. "foo.bar()" is equivalent to "foo.Util.bar()"
-	let util_completions = match namespace.env.lookup_nested_str("Util", None) {
-		LookupResult::Found(kind, _) => match kind {
-			SymbolKind::Type(typeref) => {
+	let mut util_completions = vec![];
+	for ns_env in &namespace.envs {
+		if let LookupResult::Found(kind, _) = ns_env.lookup_nested_str("Util", None) {
```

Comment:
Maybe while we here, move `"Util"` to some const

Reply:
Done